shell/utils/v-sphere.ts

```diff
@@ -101,6 +123,8 @@ class VSphereUtils {
     const { userValues, combined } = this.findChartValues(rke2Component, 'rancher-vsphere-cpi') || {};
 
     if (!combined?.vCenter?.credentialsSecret?.generate) {
+      this.resetCpiValues(userValues, downstreamName);
```

Comment:
if generate is off we should only clear username/password and keep the other settings (specifically credentialsSecret.name / configSecret.name, but also host and datacenters).

this will ensure existing clusters with existing secrets won't break and continue to support use case where users enter their own secret

Reply:
@richard-cox I'll close this PR and create a new one